README.md

```diff
@@ -10,11 +10,11 @@
 
 ## Task 1: Set Up The Project With Git
 
-* [ ] Fork the project into your GitHub user account
-* [ ] Clone the forked project into a directory on your machine
-* [ ] Create a branch called `develop` on your local build
-* [ ] Set your HEAD to your newly created `develop` branch
-* [ ] You are now ready to build this project with your preferred IDE
+* [x ] Fork the project into your GitHub user account
+* [x ] Clone the forked project into a directory on your machine
+* [x ] Create a branch called `develop` on your local build
+* [x ] Set your HEAD to your newly created `develop` branch
+* [x ] You are now ready to build this project with your preferred IDE
```

Comment:
For this markdown to work, it needs to be `* [x] some task`, with the product being:
* [x] some task
`* [x ] some task` produces this:
* [x ] some task